proposal/briansp8210/README.md

```diff
@@ -0,0 +1,63 @@
+# Pedometer
+
+## Basic information
+
+GitHub repository : [https://github.com/briansp8210/pedometer](https://github.com/briansp8210/pedometer)
+
+In modern world, people like to quantify their activities to improve quality of life. As an app installed in virtually all smart phones, pedometer becomes one of the necessary tool. In this project, I would like to follow [this elaborate tutorial](http://aosabook.org/en/500L/a-pedometer-in-the-real-world.html) and implement a simple pedometer.
+
+## Problem to solve
+
+* To implement a pedometer, we need to process the given acceleration data, fetch the portion caused by regular movement, and analyze steps count from it
+* Filter like infinite impulse response (IIR) can be used to extract user caused acceleration. It can also used to filter out effect from varied terrain
+
+### Perspective users
+
+* People who are familiar with CLI and want to get their step counts
+* Developers searching for library for couting steps given sensor data
+
+## System architecture
+
+* Users will provide the data generated from accelerometer in their phones
+* `Parser` will convert the data to a standard format, make following processing agnostic to devices
+* `Processor` will perform works like filtering, etc. to get rid of external effects and get a clear signal
+* `Calculator` will calculate step counts, distances, etc. from this signal
+* Output these results to users
+
+## API description
+
+* This project will provide a completed CLI frontend for user to provide data and get results
+* Other programs can invoke following C++ API to consume this service:
```

Comment:
Along the course of the development, please be more specific the API will look like.